aaronsekisambu/media-app-backend
# Config dev ops 167684460

## Description
This PR Setup the projet from installing dependencies, Sequelize, Setup Docker, Setup CircleCI and setup the folder structure.
## Type of change
Please select the relevant option


## How Has This Been Tested?
Please describe the tests that you ran to verify your change

## How to Manually Test
- Clone this Repo `https://github.com/aaronsekisambu/media-app-backend`.
- Run `npm install`
- Run `npm start`
- You can be able to see newly added features.
## Checklist:

## Design prototype link(s) (for UI stories)
N/A
## Implementation Plan link(s)
N/A
## Screenshots
N/A
## Pivotal Tacker
[#167684460](https://www.pivotaltracker.com/n/projects/2380790/stories/167684460)
